app/controllers/TpsController.scala

```diff
@@ -160,11 +160,24 @@ class TpsController @Inject() (actions:        Actions,
   def parseTpsPaymentsItemsForEmail(tpsPayments: TpsPayments): JsArray = {
     JsArray(tpsPayments.payments.map(nextPaymentItem =>
       toJson(TpsPaymentItemForEmail(
-        taxType           = nextPaymentItem.taxType.toString,
-        amount            = nextPaymentItem.amount.toString,
+        taxType           = getTaxTypeString(nextPaymentItem.taxType),
+        amount            = nextPaymentItem.amount.setScale(2).toString,
         transactionNumber = nextPaymentItem.chargeReference)
       )))
   }
+  
+  private def getTaxTypeString(taxType: TaxType): String = taxType match {
+    case TaxTypes.ChildBenefitsRepayments => "Child Benefits repayments"
+    case TaxTypes.Sa                      => "Self Assessment"
+    case TaxTypes.Sdlt                    => "Stamp Duty Land Tax"
+    case TaxTypes.Safe                    => "SAFE"
+    case TaxTypes.Cotax                   => "Corporation Tax"
+    case TaxTypes.Ntc                     => "Tax credit repayments"
+    case TaxTypes.Paye                    => "PAYE"
+    case TaxTypes.Nps                     => "NPS"
+    case TaxTypes.Vat                     => "VAT"
+    case _                                => taxType.toString
```

Comment:
Please avoid wild cards in pattern matching. Make it explicit (list all tax types here). It will be easier in future to maintain the code (we won't forget to update this function when introducing a new tax type).

Reply:
Ok will do 👍